Add support.md and _config.yml

WelcomePastToday opened the pull request:
Created the new support page

Commit:
Add support.md and _config.yml

ldko commented on support.md:
```diff
@@ -0,0 +1,46 @@
+---
+
+
+The **End of Term Web Archive (EOT 2024)** is a collaborative project, and we rely on the contributions and expertise of our partner institutions and the ongoing support of individuals and organizations like you! There are many ways to get involved and make a difference. Only YOU can prevent the loss of government information!
```
```suggestion
The **End of Term Web Archive (EOT)** is a collaborative project, and we rely on the contributions and expertise of our partner institutions and the ongoing support of individuals and organizations like you! There are many ways to get involved and make a difference. Only YOU can prevent the loss of government information!
```

ldko commented on support.md:
```diff
@@ -0,0 +1,46 @@
+- Submit **individual URLs** to the [URL Nomination Tool](https://digital2.library.unt.edu/nomination/eth2024/).
+- Upload **bulk seed lists** through our [GitHub repository](https://github.com/end-of-term/eot2024/tree/main/seed-lists). Bulk lists can be in .txt (one line per url) or .xlsx formats. 
+- Email bulk lists directly to us at **[eot-info@archive.org](mailto:eot-info@archive.org)**.
+- Some content is not collected by standard web harvesting tools, such as web apps and databases (eg. the eLibrary of the Federal Energy Regulatory Commission (FERC) which has thousands of dockets, reports, cases, opinions etc that are only available via search query (https://elibrary.ferc.gov/eLibrary/search)).  Help us surface **deep links** within government databases to ensure comprehensive p ...
```
```suggestion
- Some content is not collected by standard web harvesting tools, such as web apps and databases (e.g. the eLibrary of the Federal Energy Regulatory Commission (FERC) which has thousands of dockets, reports, cases, opinions etc. that are only available via search query (https://elibrary.ferc.gov/eLibrary/search).  Help us surface **deep links** within government databases to ensure comprehensive preservation. Submit .gov databases to us using our [Google form](https://forms.gle/ePGcew9X6Bt8bGLn6) or email us if you are able to comprehensively extract output URLs from .gov databases.
```

ldko commented on support.md:
```diff
@@ -0,0 +1,46 @@
+### Promote EOT on Social Media
+Spread the word about our mission by sharing information about the End of Term project on your social media channels. Follow and tag us to amplify our reach:
+- **X (formerly Twitter):** [@eotarchive](https://www.twitter.com/eotarchive)
+- **Bluesky:** [@eotarchive.org](https://bsky.app/profile/eotarchive.org) 
```
When I follow this link to Bluesky I get "Oops! Unable to resolve handle".

ldko replied:
Should the link be:
```suggestion
- **Bluesky:** [@eotarchive.org](https://bsky.app/profile/did:plc:qh5jxhakdkozbxjlivbmwtmn) 
```

ldko commented on support.md:
```diff
@@ -0,0 +1,46 @@
+- Researchers have reliable access to digital government records.
+- Future generations can learn from today’s decisions and policies.
+
+
+---
+
+
```
```suggestion
```
The horizontal rule looks odd to me there. Not sure about including it.

WelcomePastToday committed:
Update support.md

Co-authored-by: Lauren Ko <lauren.ko@unt.edu>

WelcomePastToday committed:
Update support.md

Co-authored-by: Lauren Ko <lauren.ko@unt.edu>

WelcomePastToday committed:
Update support.md

Co-authored-by: Lauren Ko <lauren.ko@unt.edu>

WelcomePastToday committed:
Update support.md

Co-authored-by: Lauren Ko <lauren.ko@unt.edu>

WelcomePastToday committed:
Update support.md

Added "For those unfamiliar with how github works, you can email bulk lists.."

WelcomePastToday committed:
Update support.md

Addition of Smokey Bear poster image.
This image may require additional formatting.

WelcomePastToday committed:
Update support.md

Linked image

WelcomePastToday committed:
Update support.md

Image USDA Smokey Bear image edit

ldko committed:
Nitpicking on capitalization
